src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +55,15 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    @Override
```

Comment:
You may apply a static modifier to `getPrintableString(Printable printables)`. A static method invoked without the need for creating an instance of a class, and the method is stored in one memory address.